### PR TITLE
Update limitations.md

### PR DESCRIPTION
**Pull request by benclarkwood**

### CODE REVIEW

File: frameworks/cassandra/docs/limitations.md [hunk 1]
```diff
@@ -37,6 +37,24 @@ The name of the data center cannot be changed after installation. `service.data_
 }
 ```
 
+## Service User
```
Comment (nickbp) [inline frameworks/cassandra/docs/limitations.md]: `Service user` to at least match with the other headers for now

**Comment by sascala** [comment]
_thank you_

File: frameworks/cassandra/docs/limitations.md [hunk 2]
```diff
@@ -37,6 +37,24 @@ The name of the data center cannot be changed after installation. `service.data_
...
+
+The default user is `nobody`. However, if the UID of `nobody` is not `65534` on the host agents, then Cassandra must be run as root.
```
Comment (nickbp) [inline frameworks/cassandra/docs/limitations.md]: "must be run as `root`"

File: frameworks/cassandra/docs/limitations.md [hunk 3]
```diff
@@ -37,6 +37,24 @@ The name of the data center cannot be changed after installation. `service.data_
...
+
+To determine the UID of `nobody`, run the command `id nobody`. The output of which is:
+```
+uid=65534(nobody) gid=65534(nobody) groups=65534(nobody)
+```
+
+If the returned UID is not `65534`, then Cassandra can be installed as root by setting the service user at install time:
+```
+"service": {
```
Comment (nickbp) [inline frameworks/cassandra/docs/limitations.md]:
```json
"service": {
  "...": "...",
  "user": "root",
  "...": "..."
}
```

**Pull request merged**